src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
```

Comment:
```suggestion
public class ColorSupplier {
    private Random random = new Random();
    
    public String getRandomColor() {
```
lets make random global, this way you will create just 1 instance or Random, instead of a new instance each time this method is invoked